test/solver/solver.cpp

```diff
@@ -177,6 +178,21 @@ struct Ir : SimpleSolverTest<gko::solver::Ir<solver_value_type>> {
 };
 
 
+struct Chebyshev : SimpleSolverTest<gko::solver::Chebyshev<solver_value_type>> {
+    static double tolerance() { return 1e7 * r<value_type>::value; }
+
+    static typename solver_type::parameters_type build_preconditioned(
+        std::shared_ptr<const gko::Executor> exec,
+        gko::size_type iteration_count, bool check_residual = true)
+    {
+        return SimpleSolverTest<gko::solver::Chebyshev<solver_value_type>>::
+            build(exec, iteration_count, check_residual)
```

Comment:
what about the foci values here?

Reply:
it use the default one as 0, 1. It might not converge. We only check whether it gives the same result as the reference like the others from random matrices.

Reply:
That's fine, I just missed the default values for the foci.